apps/web/app/settings/(settings-layout)/developer/webhooks/new/page.tsx

```diff
@@ -19,7 +32,9 @@ const Page = async () => {
       description={t("add_webhook_description", { appName: APP_NAME })}
       borderInShellHeader={true}
       backButton>
-      <NewWebhookView />
+      <Suspense fallback={<SkeletonLoader />}>
+        <NewWebhookView />
+      </Suspense>
```

Comment:
As the trpc query has suspense: true in `NewWebhookView`. we would have ot use Suspense to show correct skeleton loader